frontend/packages/console-dynamic-plugin-sdk/src/schema/console-extensions.ts

```diff
@@ -129,3 +129,7 @@ export type SupportedExtension =
  * Schema of Console plugin's `console-extensions.json` file.
  */
 export type ConsoleExtensionsJSON = SupportedExtension[];
+
+const CONSOLE_EXTENSION_PREFIX = 'console.';
```

Comment:
This could potentially still match dev console extensions like `console.topology/details/tab` and break when dev console moves to a dynamic plugin (unless we change the extension point name). Can we filter for extensions that match `SupportedExtension` types instead?

cc @christianvogt

Reply:
I'm not a fan of filtering on `console.` extension tbh.